src/quo/core.cljs

```diff
@@ -283,6 +284,9 @@
 (def url-preview quo.components.links.url-preview.view/view)
 (def url-preview-list quo.components.links.url-preview-list.view/view)
 
+;;;; Status links
+(def status-link-preview-community quo.components.internal-link-card.community.view/view)
```

Comment:
why rename this to something very confusing? please keep the name the same unless there is a clash. i.e `internal-link-card`